### PR TITLE
Migrate `react-stripe-elements` to `react-stripe-js`

### PR DESCRIPTION
**Pull request by SudharakaP**
This migrates the react-stripe-elements dependency to react-stripe-js as the former is now archived/deprecated.

I've simplified my previous PR without trying to migrate to functional components. Hope it's cleaner now. 😄 

Resolves https://github.com/opencollective/opencollective/issues/4145

cc: @Betree 

## Screencasts

Peek 2021-04-16 09-53

Peek 2021-04-16 09-55

Peek 2021-04-16 09-56

### CODE REVIEW

File: components/contribution-flow/index.js [hunk 1]
```diff
@@ -117,6 +118,7 @@ class ContributionFlow extends React.Component {
     this.state = {
       error: null,
       stripe: null,
+      elements: null,
```
Comment (Betree) [inline components/contribution-flow/index.js:121]: In this scope, calling this variable `elements` is not very explicit. Can we find a better name?

File: components/recurring-contributions/UpdatePaymentMethodPopUp.js [hunk 2]
```diff
@@ -238,6 +239,7 @@ const UpdatePaymentMethodPopUp = ({ setMenuState, contribution, onCloseEdit, loa
   const [selectedPaymentMethod, setSelectedPaymentMethod] = useState(null);
   const [loadingSelectedPaymentMethod, setLoadingSelectedPaymentMethod] = useState(true);
   const [stripe, setStripe] = useState(null);
+  const [elements, setElements] = useState(null);
```
Comment (Betree) [inline components/recurring-contributions/UpdatePaymentMethodPopUp.js:242]: Same here, we should make it clearer that this is about stripe

File: lib/errors/index.js [hunk 3]
```diff
@@ -128,6 +128,7 @@ export const generateNotFoundError = searchTerm => {
  * @returns {OCError}
  */
 export const getErrorFromGraphqlException = exception => {
+  console.log(exception);
```
Comment (Betree) [inline lib/errors/index.js:131]:
```suggestion
```

**Review by Betree — APPROVED** [review]
Thanks for simplifying, this is way easier to review :-) Some feedback about variables naming below, I haven't re-tested but the code looks good to me

File: components/contribution-flow/index.js (hunk 1 above)
Comment (SudharakaP) [reply components/contribution-flow/index.js]: Done. 👍🏽

File: components/recurring-contributions/UpdatePaymentMethodPopUp.js (hunk 2 above)
Comment (SudharakaP) [reply components/recurring-contributions/UpdatePaymentMethodPopUp.js]: Done. 😄

File: lib/errors/index.js (hunk 3 above)
Comment (SudharakaP) [reply lib/errors/index.js]: Ah, I missed removing it. 😄

**Comment by SudharakaP** [comment]
> Thanks for simplifying, this is way easier to review :-) Some feedback about variables naming below, I haven't re-tested but the code looks good to me

@Betree : Thanks for the review. Have made the changes. Let me know if you see anything else. 👍🏽

**Review by Betree — APPROVED** [review]
That looks nice, thanks for making these changes!

**Comment by SudharakaP** [comment]
> That looks nice, thanks for making these changes!

You are welcome. 😸

**Pull request merged**